### PR TITLE
i18n(frontend) caught some missing translations while translating

### PR DESCRIPTION
**Pull request by joebordes**

### CODE REVIEW

**Comment by salsakran** [comment]
@tlrobinson mind taking a look at this, specifically the use of `jt` ?

File: frontend/src/metabase/xray/components/InsightCard.jsx [hunk 1]
```diff
@@ -178,14 +178,14 @@ export class VariationTrendInsight extends Component {
 
     return (
       <InsightText>
-        It looks like this data has grown {mode}ly{" "}
+        {t`It looks like this data has grown ${mode}ly`}{" "}
```
Comment (tlrobinson) [inline frontend/src/metabase/xray/components/InsightCard.jsx]: We'll probably want to add a map like 

```javascript
const MODE_ADVERB_STRINGS = {
  "big": t`bigly`,
  "huge": t`hugely`,
  // ...
}
```

(I don't know what the actual values of `mode` can be. @sbelak?)

Then do:

```javascript
{t`It looks like this data has grown ${MODE_ADVERB_STRINGS[mode]}`}{" "}
```

**Review by tlrobinson — APPROVED** [review]
LGTM except there's one difficult to translate string I noted. I'm ok with it being done as a follow up PR unless it's easy for @joebordes to do soon.

**Comment by joebordes** [comment]
Have a look and let me know if you need any other changes

File: frontend/src/metabase/xray/components/InsightCard.jsx [hunk 2]
```diff
@@ -175,17 +175,21 @@ export class VariationTrendInsight extends Component {
 
   render() {
     const { mode } = this.props;
+    const MODE_ADVERB_STRINGS = {
+      big: t`bigly`,
```
Comment (tlrobinson) [inline frontend/src/metabase/xray/components/InsightCard.jsx]: Sorry I should have made it clear those were placeholder words, I didn't know the actual words, but I checked and they are the following:

```javascript
 const MODE_ADVERB_STRINGS = {
  increasing: t`increasingly`,
  decreasing: t`decreasingly`
}
```

I'd also move it above `export class VariationTrendInsight extends Component {` and rename it `VARIATION_MODE_ADVERB_STRINGS`

Thanks

**Review by tlrobinson — DISMISSED** [review]

**Comment by salsakran** [comment]
updated to reflect your comment, @tlrobinson

**Comment by salsakran** [comment]
thanks so much @joebordes !

**Pull request merged**